mava/components/training/losses.py

```diff
@@ -259,20 +313,20 @@ def policy_loss_fn(
                             policy_params, observations
                         )
 
-                    log_probs = network.log_prob(distribution_params, actions)
-                    entropy = network.entropy(distribution_params)
+                    log_probs = network.log_prob(distribution_params, actions) * mask
+                    entropy = network.entropy(distribution_params) * mask
```

Comment:
If you can make the masking optional so in case we don't need it we set the flag to false